std/process.d

```diff
@@ -550,6 +552,11 @@ private Pid spawnProcessImpl(in char[] commandLine,
     prepareStream(stdout, STD_OUTPUT_HANDLE, "stdout", stdoutFD, startinfo.hStdOutput);
     prepareStream(stderr, STD_ERROR_HANDLE,  "stderr", stderrFD, startinfo.hStdError );
 
+    if ((startinfo.hStdInput  != null && startinfo.hStdInput  != INVALID_HANDLE_VALUE)
+     || (startinfo.hStdOutput != null && startinfo.hStdOutput != INVALID_HANDLE_VALUE)
+     || (startinfo.hStdError  != null && startinfo.hStdError  != INVALID_HANDLE_VALUE))
+        startinfo.dwFlags = STARTF_USESTDHANDLES;
```

Comment:
Is it OK for any of the `startinfo.hStdXxx` fields to be `null` or `INVALID_HANDLE_VALUE` when `STARTF_USESTDHANDLES` is specified? The [documentation](https://msdn.microsoft.com/en-us/library/windows/desktop/ms686331%28v=vs.85%29.aspx#STARTF_USESTDHANDLES) doesn't seem to be 100% clear on this; it could be read as though they all have to point to valid, inheritable handles.